### PR TITLE
Updated French Translation.

### PR DESCRIPTION
**Pull request by Revo78**
Fixed various grammatical errors and translated other options.

### CODE REVIEW

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 1]
```diff
@@ -417,6 +417,7 @@
                 <Original>Game Logic</Original>
                 <Russian>Игровая логика</Russian>
                 <English>Game Logic</English>
+				<French>Logique de jeu</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Weird indentation you have here.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 2]
```diff
@@ -1129,7 +1130,7 @@
             <Key ID="STR_AMAE_SEARCH_AND_GARRISON_BUILDING">
                 <Original>Search and Garrison Building</Original>
                 <English>Search and Garrison Building</English>
-                <French>Cherchez et occupez un bâtiment</French>
+				<French>Chercher et occuper le batiment en garnison</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 3]
```diff
@@ -1316,6 +1317,7 @@
             <Key ID="STR_AMAE_PLAY_MESSAGE">
                 <Original>Play Radio Message</Original>
                 <English>Play Radio Message</English>
+				<French>Jouer un message radio</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 4]
```diff
@@ -1324,6 +1326,7 @@
             <Key ID="STR_AMAE_SET_FREQUENCIES">
                 <Original>Set Radio Frequencies</Original>
                 <English>Set Radio Frequencies</English>
+				<French>(TFAR)Ajuster fréquences radio</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 5]
```diff
@@ -1332,6 +1335,7 @@
             <Key ID="STR_AMAE_ROTATION_MODULE">
                 <Original>Rotate Objects</Original>
                 <English>Rotate Objects</English>
+				<French>Tourner des Objets</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 6]
```diff
@@ -1340,6 +1344,7 @@
             <Key ID="STR_AMAE_HIDE_OBJECTS">
                 <Original>Hide Objects</Original>
                 <English>Hide Objects</English>
+				<French>Cacher des Objets</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 7]
```diff
@@ -1348,6 +1353,7 @@
             <Key ID="STR_AMAE_SUPPLY_DROP">
                 <Original>Supply Drop</Original>
                 <English>Supply Drop</English>
+				<French>Largage de ravitaillement</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 8]
```diff
@@ -1356,6 +1362,7 @@
             <Key ID="STR_AMAE_EDIT_CARGO">
                 <Original>Edit Cargo</Original>
                 <English>Edit Cargo</English>
+				<French>Editer la cargaison</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 9]
```diff
@@ -1632,6 +1639,7 @@
             <Key ID="STR_AMAE_CARGO_VEHICLE_SPAWN_ERROR">
                 <Original>Error! Failed to spawn cargo vehicle</Original>
                 <English>Error! Failed to spawn cargo vehicle</English>
+				<French>Erreur! Échec de l'apparition du vehicule dans le cargaison.</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 10]
```diff
@@ -1640,6 +1648,7 @@
             <Key ID="STR_AMAE_AIRCRAFT_SPAWN_ERROR">
                 <Original>Error! Failed to spawn aircraft!</Original>
                 <English>Error! Failed to spawn aircraft!</English>
+				<French>Erreur! Échec de l'apparition du vehicule aéronautique.</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 11]
```diff
@@ -1648,6 +1657,7 @@
             <Key ID="STR_AMAE_CODE_EXECUTION_NOT_ALLOWED">
                 <Original>This module has been disabled by the mission creator.</Original>
                 <English>This module has been disabled by the mission creator.</English>
+				<French>Ce module à été désactivée par le createur de la mission.</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 12]
```diff
@@ -1656,6 +1666,7 @@
             <Key ID="STR_AMAE_NO_BUILDINGS_FOUND">
                 <Original>No buildings found!</Original>
                 <English>No buildings found!</English>
+				<French>Aucun bâtiments trouvées.</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 13]
```diff
@@ -1665,13 +1676,15 @@
                 <Original>Allocation error: Could not create a door lock logic! (occured in %1/%2 cases)</Original>
                 <English>Allocation error: Could not create a door lock logic! (occured in %1/%2 cases)</English>
                 <Russian>Ошибка выделения памяти: Не могу создать модуль логики закрытия дверей! (для %1/%2 случаев)</Russian>
+				<French>Erreur d'allocation: Échec de la creation d'une logique de verrouillage de porte! (S'est produit dans %1/%2 des cas)</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 14]
```diff
@@ -1665,13 +1676,15 @@
...
                 <Japanese>アロケーション エラー: 扉の施錠は作成されていません。 (%1/%2 の場合により)</Japanese>
                 <Chinesesimp>分配错误：无法建立锁门逻辑！（%1/%2的情况下会导致此错误）</Chinesesimp>
                 <Chinese>分配錯誤:無法建立鎖門邏輯!(%1/%2的情況下會導致此錯誤)</Chinese>
             </Key>
             <Key ID="STR_AMAE_FAILED_TO_ATTACH_CARGO">
                 <Original>Failed to attach the cargo!</Original>
                 <English>Failed to attach the cargo!</English>
+				<French>Échec de l'attachement de la cargaison.</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 15]
```diff
@@ -1680,6 +1693,7 @@
             <Key ID="STR_AMAE_CARGO_ALREADY_ATTACHED">
                 <Original>Cargo is already attached!</Original>
                 <English>Cargo is already attached!</English>
+				<French>La cargaison est déja attachée!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 16]
```diff
@@ -1688,6 +1702,7 @@
             <Key ID="STR_AMAE_NO_HELICOPTER_SELECTED">
                 <Original>No helicopter selected!</Original>
                 <English>No helicopter selected!</English>
+				<French>Aucun hélicoptère sélectionée!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 17]
```diff
@@ -1696,6 +1711,7 @@
             <Key ID="STR_AMAE_HELICOPTER_NEEDS_DRIVER">
                 <Original>The helicopter needs a driver!</Original>
                 <English>The helicopter needs a driver!</English>
+				<French>Cet hélicoptère à besoin d'un pilote!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 18]
```diff
@@ -1704,6 +1720,7 @@
             <Key ID="STR_AMAE_UNITS_NOT_ALLOWED">
                 <Original>Units are not allowed!</Original>
                 <English>Units are not allowed!</English>
+				<French>Les unités ne sont pas admises!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 19]
```diff
@@ -1712,6 +1729,7 @@
             <Key ID="STR_AMAE_TFAR_HAS_TO_BE_LOADED">
                 <Original>This module requires Task Force Radio!</Original>
                 <English>This module requires Task Force Radio!</English>
+				<French>Ce module requiert Task Force Radio!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 20]
```diff
@@ -1857,14 +1875,15 @@
             <Key ID="STR_AMAE_SELECT_NON_PLAYER_UNITS">
                 <Original>Select non player units only!</Original>
                 <English>Select non player units only!</English>
+				<French>Sélectionner seulement les unités non joueur!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 21]
```diff
@@ -1984,6 +2003,7 @@
             <Key ID="STR_AMAE_NO_TELEPORT_DESTINATION">
                 <Original>Destination no longer exists!</Original>
                 <English>Destination no longer exists!</English>
+				<French>La déstination n'existe plus!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 22]
```diff
@@ -1992,6 +2012,7 @@
             <Key ID="STR_AMAE_OBJECT_ALREADY_TELEPORTER">
                 <Original>Object is already a teleporter!</Original>
                 <English>Object is already a teleporter!</English>
+				<French>L'objet est deja un téleporteur!</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 23]
```diff
@@ -2034,6 +2055,7 @@
             <Key ID="STR_AMAE_NONE">
                 <Original>None</Original>
                 <English>None</English>
+				<French>Aucun</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 24]
```diff
@@ -2043,6 +2065,7 @@
             <Key ID="STR_AMAE_CREATED_TELEPORT">
                 <Original>Created teleporter '%1'</Original>
                 <English>Created teleporter '%1'</English>
+				<French>Téléporteur crée '%1'</French>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation...

**Comment by Revo78** [comment]
Fixed the indentation. Can you please review it now.

**Comment by Revo78** [comment]
Weird, on Sublime Text, my identation is correct but i have allowed edits now.

**Comment by CreepPork** [comment]
There's a big surge of changes and it's due to whitespace changes, so I'm guessing you might have changed the line breaks or just added a new whitespace.

**Comment by CreepPork** [comment]
The indentation is overall fine. If @oOKexOo you think you can fix the surge, then feel free.

**Comment by oOKexOo** [comment]
@CreepPork 
`regex + git diff` tells me otherwise.

**Pull request merged**